bindings/nodejs/upgrade.md

```diff
@@ -1,3 +1,11 @@
+# Upgrade to v0.53
```

Comment:
The next version of nodejs will be `0.48`:

https://github.com/apache/opendal/blob/3cfce47652724b21db33f297c2f84d54cf18e17a/bindings/nodejs/package.json#L4